build.sbt

```diff
@@ -6,14 +6,16 @@ name               := "scala-swing"
 
 version            := "2.0.4-SNAPSHOT"
 
-scalacOptions      ++= Seq("-deprecation", "-feature")
+scalacOptions      ++= Seq("-deprecation", "-feature", "-Xlint")
+
+crossScalaVersions := Seq("2.12.8", "2.13.0-M5")
```

Comment:
What happened to Scala 2.11 - has it never been specified as `crossScalaVersions`? If we add it, we also need scala-2.11 specific source directory. We can probably use a symlink to scala-2.12. Or wasn't there a way to specify something like `scala.2.12-` (2.12. or lower; I have a vague memory)

Reply:
Actually, `crossScalaVersions` is not needed, as the module-plugin creates that from `scalaVersionsByJvm`

Reply:
Thanks, I didn't understand that.